src/Aspire.Dashboard/Model/ResourceOutgoingPeerResolver.cs

```diff
@@ -96,9 +96,13 @@ bool TryMatchResourceAddress(string value, [NotNullWhen(true)] out string? name)
         {
             foreach (var (resourceName, resource) in resources)
             {
-                foreach (var service in resource.Services)
+                foreach (var service in resource.Urls)
                 {
-                    if (string.Equals(service.AddressAndPort, value, StringComparison.OrdinalIgnoreCase))
+                    var uri = new Uri(service.Url);
```

Comment:
`Uri.TryCreate` is used ealier. Should be tolerant of a badly formed Url here as well.

Reply:
Done, stored the uri in the view model